src/Nethermind/Nethermind.BeaconNode.Peering/PeerDetails.cs

```diff
@@ -14,37 +14,32 @@
 //  You should have received a copy of the GNU Lesser General Public License
 //  along with the Nethermind. If not, see <http://www.gnu.org/licenses/>.
 
-using Nethermind.Core2.Types;
+using System.Diagnostics;
+using Nethermind.Core2.P2p;
 
 namespace Nethermind.BeaconNode.Peering
 {
-    public class PeerSyncStatus
+    public class PeerDetails
     {
-        private Slot _highestPeerSlot;
-
-        public Slot HighestPeerSlot
+        public PeerDetails(string id)
         {
-            get => _highestPeerSlot;
+            Id = id;
         }
+        
+        public string Id { get; }
+        
+        public DialDirection DialDirection { get; private set; }
```

Comment:
Check out Session, Peer, ConnectionDirection, SessionState in Eth1, maybe we can reuse some of it?

Reply:
Made things compatible where possible, e.g. direction is the same, but states will be different (different protocol).  While the design can be reused, I think the protocols are different enough that we may not be able to directly reuse the same objects.